runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/ParDoTranslation.java

```diff
@@ -496,6 +497,25 @@ private static SdkFunctionSpec toProto(ViewFn<?, ?> viewFn) {
         .build();
   }
 
+  private static <T> ParDoPayload getParDoPayload(AppliedPTransform<?, ?, ?> transform)
+      throws IOException {
+    return PTransformTranslation.toProto(
```

Comment:
This feels really circuitous. I think I understand why you're doing it, but a comment probably wouldn't be out of place here.